bootstrap/kubeadm/docs/external-etcd.md

```diff
@@ -106,4 +129,8 @@ spec:
     ... # other clusterConfiguration goes here
 ```
 
-Create your cluster as normal!
+Create your workload cluster as normal. The new workload cluster should use the configured external etcd nodes instead of creating co-located etcd Pods on the control plane nodes.
+
+## Additional Notes/Caveats
+
+* Depending on the provider, additional changes to the workload cluster's manifest may be necessary to ensure the new CAPI-managed nodes have connectivity to the existing etcd nodes. For example, on AWS you will need to leverage the `additionalSecurityGroups` field on the AWSMachine and/or AWSMachineTemplate objects to add the CAPI-managed nodes to a security group that has connectivity to the existing etcd cluster. Other mechanisms exist for other providers.
```

Comment:
if someone likes to put their etcd cluster behind a LB this has the benefit of kubeadm not knowing the whole list of external member SANs. in theory this would allow the etcd cluster to grow / change without kubeadm knowing. unclear to me weather `--etcd-servers` for kube-apiservers can including only that LB though.

we don't have this in the kubeadm docs.

Reply:
It sounds like maybe we can omit a discussion of using an LB for etcd here until it has been added to other documentation. Is there an issue somewhere for addressing this in the `kubeadm` docs? Maybe I can help in some way.

Reply:
> It sounds like maybe we can omit a discussion of using an LB for etcd here

yes

>  Is there an issue somewhere for addressing this in the kubeadm docs

no, i don't think anyone has tested this on our side.

> Maybe I can help in some way.

it's very low priority unless there is more demand. before documenting it, it needs an e2e test too using kinder.